packages/datadog-plugin-sharedb/src/index.js

```diff
@@ -0,0 +1,165 @@
+'use strict'
+
+const MessagesAwaitingResponse = new WeakMap()
+const READABLE_ACTION_NAMES = {
```

Comment:
Any way to avoid having to maintain a static list?

Reply:
If not, any link that can be added in a comment of where the list comes from?

Reply:
This isn't exposed from sharedb, I can try to add a link I suppose.

Reply:
Done!